osquery/tables/system/windows/processes.cpp

```diff
@@ -96,14 +119,41 @@ void genProcess(long pid, QueryData& results_data) {
       r["egid"] = "-1";
       r["sgid"] = "-1";
 
-      r["wired_size"] = "0";
-      r["resident_size"] = "0"; // Populate with WorkingSetSize (VT_BSTR)
-      r["phys_footprint"] = "0";
-
-      r["user_time"] = "0";
-      r["system_time"] = "0";
-      r["start_time"] = "0";
+      // We pre-populate the info, in the event the API calls fail.
+      r["user_time"] = "-1";
+      r["system_time"] = "-1";
+      r["start_time"] = "-1";
+      r["wired_size"] = "-1";
+      r["resident_size"] = "-1";
+      r["phys_footprint"] = "-1";
+
+      PROCESS_MEMORY_COUNTERS pmc;
+      auto ret = GetProcessMemoryInfo(hProcess, &pmc, sizeof(pmc));
+      if (ret != 0) {
+        r["wired_size"] = BIGINT(pmc.PagefileUsage);
+        r["resident_size"] = BIGINT(pmc.WorkingSetSize);
```

Comment:
Does this give current RSS? Is `PrivatePageCount * PageSize` more honest on a poll-by-poll?